examples/examples_nodejs_test.go

```diff
@@ -173,9 +173,8 @@ func TestAccAlbNew(t *testing.T) {
 			// RunUpdateTest: true,
 			EditDirs: []integration.EditDir{
 				{
-					Dir:             "step2",
-					Additive:        true,
-					ExpectNoChanges: true,
```

Comment:
Why does this change?

Reply:
Subnets are now a computed field so changes outside of the RouteTable will cause diffs